covid_graphs/covid_graphs/bayesian.py

```diff
@@ -0,0 +1,88 @@
+# Code inspired by Vladimír Boža
+
+import random
+from pathlib import Path
+
+import click
+import click_pathlib
+import matplotlib.pyplot as plt
+import numpy as np
+import pymc3 as pm
+
+from . import country_report
+from .country_graph import CountryGraph, GraphType
+from .fit_atg_model import AtgModelFit
+from .formula import FittedFormula
+from .predictions import CountryPrediction, PredictionEvent
+
+
+def fit_bayesian_model(cases: np.ndarray):
+    base_length = 800
+    with pm.Model() as model:  # noqa: F841
```

Comment:
`with pm.Model():`

Reply:
Actually the whole construct seems a bit fishy, do you know why do we need the `with` line (what does it do)? Anyway, this is experimental functionality so we don't have to solve it.

Reply:
`with pm.Model():` doesn't work. The authors explain it here:
https://nbviewer.jupyter.org/github/CamDavidsonPilon/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers/blob/master/Chapter2_MorePyMC/Ch2_MorePyMC_PyMC3.ipynb#Model-Context

I don't think we have a lot of freedom here.